metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -11,14 +11,14 @@
   "For a given provider and entry-titles in collection-identifier, returns list of collection concept ids"
   [provider-id collection-identifier]
   (when-let [entry-titles (get collection-identifier :entry-titles)]
-      (let [provider (-> (h/get-provider provider-id)
-                         providers/dbresult->provider)
-            t (h/get-provider-collection-tablename provider)]
-        (for [entry-title entry-titles
-              result (h/query [(format "select concept_id from metadata_db.%s where
-                                        entry_title = ?" t)
-                               entry-title])]
-          (:concept_id result)))))
+    (let [provider (-> (h/get-provider provider-id)
```

Comment:
Quick note on the threading: kibit will fail that unless there are three levels getting passed. Quick fix:
```clj
(-> provider-id
    (h/get-provider)
    providers/dbresult->provider)
```